testsuite/integration/clustering/src/test/java/org/jboss/as/test/clustering/TopologyChangeListenerBean.java

```diff
@@ -77,7 +77,7 @@ public void establishTopology(String containerName, String cacheName, long timeo
                     topology = dist.getCacheTopology();
                     members = getMembers(topology);
                 }
-                logger.infof("Cache %s/%s successfully established view %s within %d ms. Topology id = %d", containerName, cacheName, expectedMembers, now - start, topology.getTopologyId());
+                logger.debugf("Cache %s/%s successfully established view %s within %d ms. Topology id = %d", containerName, cacheName, expectedMembers, now - start, topology.getTopologyId());
```

Comment:
I think we need to keep these at INFO since they often help debugging. @pferraro would you agree?

Reply:
yes - please leave this one as is.